src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1529,6 +1530,30 @@ public function getContent($asResource = false)
         return $this->content;
     }
 
+    /**
+     * Return request body content as a json decoded array.
+     *
+     * @return array
+     *
+     * @throws JsonException|\LogicException
+     */
+    public function getJson()
```

Comment:
JSON is a text format, I wouldn't expect the `getJson()` method to return an array or anything else but a string, although I'm not sure that another name fits it... `getJsonDecoded()` perhaps?

Reply:
I agree with you, but getJson() returning a string is fine IMO